pyo3-macros-backend/src/pyclass.rs

```diff
@@ -1743,20 +1743,7 @@ fn impl_pytypeinfo(cls: &syn::Ident, attr: &PyClassArgs, ctx: &Ctx) -> TokenStre
         quote! { ::core::option::Option::None }
     };
 
-    #[cfg(feature = "gil-refs")]
-    let has_py_gil_ref = quote! {
-        #[allow(deprecated)]
-        unsafe impl #pyo3_path::type_object::HasPyGilRef for #cls {
-            type AsRefTarget = #pyo3_path::PyCell<Self>;
-        }
-    };
-
-    #[cfg(not(feature = "gil-refs"))]
-    let has_py_gil_ref = TokenStream::new();
-
     quote! {
-        #has_py_gil_ref
```

Comment:
I take it `TokenStream::new()` was just a placeholder to satisfy the compiler?

Reply:
Yep it expands to no tokens :)